### ISSUE TITLE
No way to see response error

### ISSUE THREAD

**Issue by anatoliyv** [uploadcare/uploadcare-ios#30]
When uploading a big files, if file size is big enough and can not be supported by by subscription plan it's impossible to see error message. Response I got from a server:
```
POST https://upload.uploadcare.com/base/
...
File is too large.
```

Update:
Here is how error looks in SDK in completion handler:
```
(lldb) po error
▿ Optional<Error>
  - some : Error Domain=NSCocoaErrorDomain Code=3840 "JSON text did not start with array or object and option to allow fragments not set." UserInfo={NSDebugDescription=JSON text did not start with array or object and option to allow fragments not set.}
```

**Comment by rsedykh** [comment]
Hi, @anatoliyv! We've just released new uploadcare-swift integration, [0.1.0-alfa](https://github.com/uploadcare/uploadcare-swift/releases/tag/0.1.0-alpha). I hope you can check it out. Feedback is very welcome. If you have any feature requests or questions, shoot!

**Comment by rsedykh** [comment]
Btw, I forgot to mention that this issue is fixed in the new library.
